src/repository-added.ts

```diff
@@ -26,7 +26,7 @@ async function repositoryAdded(context: Context) {
                     enabled: true
                   };
                   
-      const licenseTask = new LicenseTask(AppConfig, licfg, repo);
+      const licenseTask = new LicenseTask(AppConfig, licfg, repo, undefined);
```

Comment:
like before I wouldn't use really `undefined` as a state. Normally, when you call `new LicenseTask(AppConfig, licfg, repo)` -> this already results in organization being undefined. 
Except TS is now blocking it somehow :D

Reply:
yeah this parameter is not optional TS just knows that is either a string or undefined, so you cannot call the constructor without it :)